builtin/game/item.lua

```diff
@@ -147,6 +147,23 @@ function core.wallmounted_to_dir(wallmounted)
 	return wallmounted_to_dir[wallmounted]
 end
 
+function core.dir_to_yaw(dir)
+	if dir.z < 0 then
+		return math.pi - math.atan(dir.x / dir.z)
+	elseif dir.z > 0 then
+		return -math.atan(dir.x / dir.z)
+	elseif dir.x < 0 then
+		return math.pi
+	else
+		return 0
+	end
+end
+
+function core.yaw_to_dir(yaw)
+	local y = yaw + (math.pi / 2)
```

Comment:
'y' is present as the vertical component of 'dir' below and there's no reason this has to be 'y' when 'yaw' can be used instead. It's not particularly confusing but it makes the code a little more effort to mentally process when you first look at it.
Sorry, pedantic request.